docs/markdown/Using Pants/concepts/targets.md

```diff
@@ -216,6 +217,23 @@ To reset any modified defaults, simply override with the empty dict:
     __defaults__(all={})
 ```
 
+
+Extending field defaults
+------------------------
+
+To add to a default value rather than replacing it, the current default value for a target field is available in the BUILD file using `<target>.<field>.default`. This allows you to augment a field's default value with much more precision. As an example, if you want to make the default sources for a `python_sources` target to work recursively you may specify a target augmenting the default sources field:
```

Comment:
Do you think this is obvious that this would let you extend the values that are set with `__defaults` as well? It is for me, but I am not sure if that's the case for others. Maybe add a sentence in the end to say that "This approach would also let you extend the list of default values that have been set with the `__defaults__` builtin"?

Reply:
yea.. I was considering that. Since we are in the chapter of `__defaults__` and I'd like it to remain clear that `__defaults__` provides new defaults for target fields, there really shouldn't be a question to clarify here (seeing it sort of as an implementation detail), as we're referring to a fields default value it shouldn't matter how the field got it. Caveat is that there are fields that may have dynamic defaults, and this doesn't work for those unfortunately. That might be a thing to call out, but I'm not even sure how widely used dynamic defaults are atm, and seeing that it may even go down over time I'd rather not focus on it too much :) 

So, I'm open to adding it, if it is something we likely need to clarify. Holding for more input on that :)